client/jetpack-onboarding/disclaimer.jsx

```diff
@@ -0,0 +1,28 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'gridicons';
+
+const JetpackOnboardingDisclaimer = ( { translate } ) => (
+	<p className="jetpack-onboarding__disclaimer">
+		<Gridicon icon="info-outline" size={ 18 } />
+		{ translate(
+			'By continuing, you agree to our {{link}}fascinating terms and conditions{{/link}}.',
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'By checking out, you agree to our {{link}}fascinating terms and conditions{{/link}}.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).